### PR TITLE
WIP: Add category and product business logic classes

### PR DESCRIPTION
**Pull request by cmatheny**
test pull request

### CODE REVIEW

**Comment by octocat-jedi** [comment]
Can one of the admins verify this patch?

**Comment by octocat-jedi** [comment]
test this please

**Comment by tjkemper** [comment]
test this please

**Comment by tjkemper** [comment]
test this please

**Comment by tjkemper** [comment]
test this please

**Comment by tjkemper** [comment]
test this please

**Comment by tjkemper** [comment]
test this please

**Comment by octocat-jedi** [comment]
test this please

**Comment by octocat-jedi** [comment]
test this please

**Comment by octocat-jedi** [comment]
test this please

**Comment by octocat-jedi** [comment]
test this please

**Comment by octocat-jedi** [comment]
test this please

File: src/main/java/com/ims/controllers/rest/CategoryAPI.java [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+	
+	@RequestMapping(method=RequestMethod.GET, value="/get-category")
+	public Category getCategory(int categoryID) {
+		return CategoryLogic.getCategory(categoryID);
+	}
+	
+	@RequestMapping(method=RequestMethod.GET, value="/get-all-categories")
+	public ResponseEntity<Category> getAllCategories() {
+		CategoryLogic.getAllCategories();
+		return ResponseEntity.status(HttpStatus.I_AM_A_TEAPOT).body(category);
```
Comment (tjkemper) [inline src/main/java/com/ims/controllers/rest/CategoryAPI.java]: Compilation Error: Cannot find symbol `category`

**Comment by tjkemper** [comment]
@cmatheny we edited the `Jenkinsfile` in the `environVar` branch to accommodate the new environment variables **IMS_USER**, **IMS_PASS**, and **IMS_URL**

This branch is failing because it does not have the updated `Jenkinsfile`

**Comment by tjkemper** [comment]
Updated this branch with new `Jenkinsfile` from the `master` branch

**Review by sharangw — APPROVED** [review]
Looks Noice

**Pull request merged**